### PR TITLE
rename some variable

### PR DESCRIPTION
**Pull request by Frustrated-Programmer**
Changed names of values for better readability
Tabbed in areas that wasn't formatted properly

### CODE REVIEW

File: Platform.js [hunk 1]
```diff
@@ -55,542 +55,542 @@ var fElapsed = 0;
...
-    this.topSpeed = 3;
-    this.v = 0;
-    this._v = 0;
-    this.x = x;
-    this.y = y;
+var Entity = function(x, y, team, flag){
+	this.RAM = [];
+	this.angle = 0; 
+	this.acceleration = 0.05;
+	this.teamWith = team;
```
Comment (NovaSagittarii) [inline Platform.js]: `this.teamWith` too wordy

File: Platform.js [hunk 2]
```diff
@@ -55,542 +55,542 @@ var fElapsed = 0;
...
+	this.msgCD = 0;
+	this.regen = 0;
+	this.restart = 100;
+	this.safe = true;
+	this.stamina = 0.4*this.max;
+	this.targetAngle = 0; // target angle
+	this.temp = [];
+	this.topSpeed = 3;
+	this.currentSpeed = 0;
+	this.targetSpeed = 0;
```
Comment (NovaSagittarii) [inline Platform.js]: re-alphabetize

**Comment by NovaSagittarii** [comment]
The indentation is formatted correctly, it just uses 4 spaces rather than a tab.

File: Platform.js (hunk 1 above)
Comment (Frustrated-Programmer) [reply Platform.js]: Nobody would guess alignment is the team you are with...
Not until they go though the code :P

File: Platform.js (hunk 2 above)
Comment (Frustrated-Programmer) [reply Platform.js]: oh i never noticed you have the variables alphatized

File: Platform.js (hunk 1 above)
Comment (Frustrated-Programmer) [reply Platform.js]: perhaps would `this.myTeam`, `this.team` or `this.side` work better?

File: Platform.js (hunk 2 above)
Comment (Frustrated-Programmer) [reply Platform.js]: fixed

**Comment by NovaSagittarii** [comment]
o my goodness i somehow forgot about this and now there are a bunch of merge conflicts aaa

**Comment by NovaSagittarii** [comment]
i am sorry for being so closed minded in the past but well... it's merged now
